web/client/plugins/BackgroundSelector.jsx

```diff
@@ -88,30 +92,36 @@ const backgroundSelector = createSelector([
   * }
   */
 
-const BackgroundSelectorPlugin = connect(backgroundSelector, {
-    onPropertiesChange: changeLayerProperties,
-    onToggle: toggleControl.bind(null, 'backgroundSelector', null),
-    onLayerChange: setControlProperty.bind(null, 'backgroundSelector'),
-    onStartChange: setControlProperty.bind(null, 'backgroundSelector', 'start'),
-    onAdd: addBackground,
-    onRemove: removeNode,
-    onBackgroundEdit: backgroundEdited,
-    updateNode,
-    onUpdateThumbnail: updateThumbnail,
-    removeBackground,
-    clearModal: clearModalParameters,
-    addBackgroundProperties,
-    onRemoveBackground: confirmDeleteBackgroundModal,
-    setCurrentBackgroundLayer
-}, (stateProps, dispatchProps, ownProps) => ({
-    ...stateProps,
-    ...dispatchProps,
-    ...ownProps,
-    thumbs: {
-        ...thumbs,
-        ...ownProps.thumbs
-    }
-})
+const BackgroundSelectorPlugin =
```

Comment:
Hi, i think this little refactor to recompose utilities was not so necessary.

Reply:
I only added hasCatalog property calculation from props.